README.md

```diff
@@ -19,62 +30,82 @@ This lib proxies through the OS resolution mechanism which may provide further c
 
 ## Objectives
 
-  * Async requests - dns resolve vs lookup
-  * Fast - local in-app memory cache lookup
-  * Fresh - periodically (frequently) updated
-  * Constant DNS load/latency vs random load/variable latency
-  * Providing statistics and introspection
+- Async requests - dns resolve vs lookup
+- Fast - local in-app memory cache lookup
+- Fresh - periodically (frequently) updated
+- Constant DNS load/latency vs random load/variable latency
+- Providing statistics and introspection
 
 ## Requirements
 
-ECMAScript module (esm), not native esm/.mjs with package.json type: module, requires esm
-
 Node 14+
 
 ## Getting started
 
 ```console
-npm i -S axios-cached-dns-resolve
+npm i -S @cryptotaxcalculator/axios-cached-dns-resolve
 ```
 
 # Usage
 
 ```javascript
-  import { registerInterceptor } from 'axios-cached-dns-resolve'
+import { registerInterceptor } from "@cryptotaxcalculator/axios-cached-dns-resolve";
 
-  const axiosClient = axios.create(config)
-
-  registerInterceptor(axiosClient)
+const axiosClient = axios.create(config);
 
+registerInterceptor(axiosClient);
 ```
-Use axiosClient as normal
 
+Use axiosClient as normal
 
 ## Configuration
 
 ```javascript
 const config = {
-  disabled: process.env.AXIOS_DNS_DISABLE === 'true',
```

Comment:
This is the main change in this file - we've changed the config (to account for a redis cache option as well as a `lru-cache` option)